frontend/src/storybook.test.tsx

```diff
@@ -1,58 +1,93 @@
-import initStoryshots, { Stories2SnapsConverter } from '@storybook/addon-storyshots';
-import * as rtl from '@testing-library/react';
+import 'vitest-canvas-mock';
```

Comment:
this is the replacement for addon-storyshots which doesn't work with vite and also is deprecated.
used this guide: https://storybook.js.org/docs/writing-tests/storyshots-migration-guide

now it generates one file for all snapshots which is not ideal. I've tried to generate individual files but for some reason vitest `toMatchFileSnapshot` function doesn't format the output properly when generating snapshots